web/packages/core/src/shadow-template.ts

```diff
@@ -275,12 +275,20 @@ ruffleShadowTemplate.innerHTML = `
                 transform: rotate(360deg);
             }
         }
+        #ruffle_virtual_keyboard {
```

Comment:
Personally I don't see a reason to call everything "Ruffle" - It doesn't add any meaning and only lengthens names unnecessarily. So how about staying just with `#virtual-keyboard`?

By the way, hopefully this will be extracted to a standalone CSS file soon (once `ruffle-core` will be bundled using Webpack, I'm working on this...). Then Stylelint will enforce following `kebab-case` pattern,

Reply:
Ok. This file currently uses kebab-case, camelCase, and snake_case, so i was unsure what convension to follow. Admittedly I think I added camelCase when adding the preloader, but the other two were there for a while. The ruffle part of the id was a remnant of before I added this to the shadow template, even with Javascript, as I added it to the document in the first version of this PR and used that I'd assuming it would not conflict with preexisting elements. I'll remove it.